assets/css/style.css

```diff
@@ -120,8 +119,50 @@ q{
     text-align: center;
 }
 legend{
-    color: #f5f0f0;
+    color: #f01111;
     font-weight: bold;
     font-style: oblique;
     font-size: 30px;
+}
+.home0{
+    display: grid;
+    grid-template-columns: auto 60%;
+    align-items: center;
+    margin: 0;
+}
+.home1{
+    align-items: center;
+    min-height: auto;
+}
+.home2{
+    width: 500px;
```

Comment:
make height and width 100%

Reply:
when i assigned the image with the width and height of 100%